test/unit/Set-CloudEventJsonData.Tests.ps1

```diff
@@ -3,9 +3,9 @@
 #  SPDX-License-Identifier: Apache-2.0
 # **************************************************************************
 
-Describe "Add-CloudEventJsonData Function Tests" {
-   Context "Adds Json Data" {
-      It 'Adds json data with depth 1' {
+Describe "Set-CloudEventJsonData Function Tests" {
+   Context "Sets Json Data" {
+      It 'Sets json data with depth 1' {
```

Comment:
Looks like default depth is 1 - is this documented somewhere or general knowledge in PS community?

Reply:
No, the default is 3. Added description and of the `Depth` parameter help to address this comment. The test here covers the conversion of a hashtable with depth 1.

Reply:
Thx, can we please be explicit and consistent in the test cases then too and use `-Depth n` everywhere? That is actually what threw me off during the review and why I was expecting `1` the be the default.